### PR TITLE
Added Linters

### PR DESCRIPTION
**Pull request by BHObserver**
In this branch -

- Added linters
- Modified html file
- Changed the color of text
- Fixed linter problems

### CODE REVIEW

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @BHObserver , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Linters are passing 👍 
- Descriptive pr👍 

 




 

## Required Changes ♻️

- [ ] Kindly consider making your repository look professional by checking the below points 👇 

- README file should be descriptive? AKA: Microverse [README template](https://github.com/microverseinc/readme-template) should be used.


1. All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your repo.
2. The only optional section is FAQ.
3. you should delete the live demo section if you decide not to deploy the application.
4. Make sure you add the section Future Features, including 1-3 potential features to be added.
5. you should remove the section description, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." 
doing so will not only make your repo look professional but also descriptive 
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -27,23 +37,101 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [hello_microverse_project] <a name="about-project"></a>
+# 📖 [Hello-Microverse-Project] <a name="about-project"></a>
+
+
+
+**[Hello_Microverse_Project]** is a single webpage.
```
Comment (wayungi) [inline README.md]: - [ OPTIONAL] Kindly offer a brief but wholesome descriptin of the project. 😄

File: README.md [hunk 2]
```diff
@@ -27,23 +37,101 @@
...
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
```
Comment (wayungi) [inline README.md:42]: - [ ] For all sections that are included in the readme, please add a body under the title. adding titles alone does not tell what is included under these titles 👍🏾

File: README.md [hunk 3]
```diff
@@ -27,23 +37,101 @@
...
+
+
+### Prerequisites
+
+
+### Setup
+
+
+### Install
+
```
Comment (wayungi) [inline README.md]: - [ ] Kindly add content under these titles. You can refer to the [original template ](https://github.com/microverseinc/readme-template)to get an idea of what you should write under these sub titles .

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @BHObserver , 

image

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] Please always delete the last descriptive section of the readme. It only serves to guide you when adding a Licence to your project and should be deleted once the licence is added.
delete


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @BHObserver , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**